src/sdk/bucketspace_fs.js

```diff
@@ -314,14 +316,30 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
     }
 
 
-    new_bucket_defaults(account, { name, tag, lock_enabled, force_md5_etag }, create_uls, bucket_storage_path) {
+    async new_bucket_defaults(account, { name, tag, lock_enabled, force_md5_etag }, create_uls, bucket_storage_path) {
```

Comment:
new_XX_defaults is typically a sync function, not async. if you need to fetch anything async for the defaults we usually do that before the call and pass it in.

Reply:
Thanks, It will be changed.